core/config/project_settings.cpp

```diff
@@ -1259,16 +1259,19 @@ ProjectSettings::ProjectSettings() {
 	CRASH_COND_MSG(singleton != nullptr, "Instantiating a new ProjectSettings singleton is not supported.");
 	singleton = this;
 
+	GLOBAL_DEF("application/config/auto_accept_quit", true);
```

Comment:
IIRC the settings are shown in their order of definition in the editor. This is a very niche one which should not come before `application/config/name` IMO. It would be better suited next to `application/config/quit_on_go_back` which seems related.